airbyte-integrations/connectors/source-postgres/src/testFixtures/java/io/airbyte/integrations/source/postgres/PostgresTestDatabase.java

```diff
@@ -22,6 +22,7 @@ public class PostgresTestDatabase extends
   public static enum BaseImage {
 
     POSTGRES_16("postgres:16-bullseye"),
+    POSTGRES_13("postgres:13-bullseye"),
```

Comment:
Can we replace 12 with 13, if we really want 13? Otherwise if 12 is OK, let's use 12.

Reply:
sounds good! fixed